WMIAdventure/backend/WMIAdventure_backend/IngameUsers/migrations/0004_auto_20210602_1210.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2 on 2021-06-02 10:10
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('IngameUsers', '0003_auto_20210514_0253'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='semester',
+            old_name='semesterNumber',
+            new_name='semester_number',
+        ),
+        migrations.RenameField(
+            model_name='userprofile',
+            old_name='displayedUsername',
+            new_name='displayed_username',
+        ),
+    ]
```

Comment:
Edytowanie migracji które już zostały zastosowane na serwerze to nie jest dobry pomysł. Powinieneś stworzyć nową, która zmienia nazwy tych pól w tabeli, inaczej ta migracja w ogóle nie zostanie zastosowana,